examples/with-edgedb/components/Header.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+
+const Header: React.FC = () => {
+  const router = useRouter()
+  const isActive: (pathname: string) => boolean = (pathname) =>
```

Comment:
The `isActive` check is a bit more verbose and is shown here https://github.com/vercel/next.js/blob/bc40c0b530f9a2cde76e7f1537c9f4abe66dd69d/examples/active-class-name/components/ActiveLink.js so might be good to leave out of this example

Reply:
Yeah probably overkill 😅